input/ignoreWarnings.txt

```diff
@@ -219,3 +219,14 @@ The definition for the Code System with URI 'urn:oid:2.16.840.1.113883.6.238' do
 
 # SECTION 2 - INFORMATION - The following messages are generated from code systems and value sets currently in 'draft' status within the inherited/base resource snapshot.
 Reference to draft CodeSystem http://terminology.hl7.org/CodeSystem/discharge-disposition|1.0.0
+
+# The following are confirmed for suppression based on US Core:
+ERROR:	The code provided (urn:oid:2.16.840.1.113883.6.238#1586-7) was not found in the value set 'Detailed Race' (http://hl7.org/fhir/us/core/ValueSet/detailed-race|6.1.0), and a code from this value set is required: The provided code 'urn:oid:2.16.840.1.113883.6.238#1586-7 ('Shoshone')' was not found in the value set 'http://hl7.org/fhir/us/core/ValueSet/detailed-race|6.1.0'
+ERROR:	The code provided (urn:oid:2.16.840.1.113883.6.238#2036-2) was not found in the value set 'Detailed Race' (http://hl7.org/fhir/us/core/ValueSet/detailed-race|6.1.0), and a code from this value set is required: The provided code 'urn:oid:2.16.840.1.113883.6.238#2036-2 ('Filipino')' was not found in the value set 'http://hl7.org/fhir/us/core/ValueSet/detailed-race|6.1.0'
+ERROR:	The code provided (urn:oid:2.16.840.1.113883.6.238#2184-0) was not found in the value set 'Detailed ethnicity' (http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity|6.1.0), and a code from this value set is required: The provided code 'urn:oid:2.16.840.1.113883.6.238#2184-0 ('Dominican')' was not found in the value set 'http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity|6.1.0'
+ERROR:	The code provided (urn:oid:2.16.840.1.113883.6.238#2148-5) was not found in the value set 'Detailed ethnicity' (http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity|6.1.0), and a code from this value set is required: The provided code 'urn:oid:2.16.840.1.113883.6.238#2148-5 ('Mexican')' was not found in the value set 'http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity|6.1.0'
+
+# This will have to be suppressed for ballot QA as an update to the extension pack is needed to correct it. This tracker should be referenced in the suppression message https://jira.hl7.org/browse/FHIR-46625:
```

Comment:
Rewrite the suppression message. 
Change from "This will have to be suppressed for ballot QA as an update to the extension pack is needed to correct it. This tracker should be referenced in the suppression message https://jira.hl7.org/browse/FHIR-46625:"
to something like this "This error is suppressed for this ballot publication only. The error will be resolved once the cqf-criteriaReference extension in the Extension Pack will be updated to allow it be used on a Reference. See tracker "https://jira.hl7.org/browse/FHIR-46625".